src/asciidoctor-kroki.js

```diff
@@ -43,14 +43,23 @@ const processKroki = (processor, parent, attrs, diagramType, diagramText, contex
   if (subs) {
     diagramText = parent.applySubstitutions(diagramText, parent.$resolve_subs(subs))
   }
-  const role = attrs.role
   const blockId = attrs.id
   const title = attrs.title
   const target = attrs.target
   const format = attrs.format || 'svg'
+  let role = attrs.role
+  if (role) {
+    if (format) {
+      role = `${role} ${format} kroki`
```

Comment:
I feel like the class name "png" or "svg" is too broad, what about `kroki-format-${format}`?

Reply:
Works for me! You are right